Cargo.toml

```diff
@@ -5,8 +5,19 @@ description = "Minimal non-blocking I/O layer"
 keywords = ["await", "futures", "IO"]
 license = "MIT OR Apache-2.0"
 name = "nb"
-repository = "https://github.com/japaric/nb"
+repository = "https://github.com/rust-embedded/nb"
+homepage = "https://github.com/rust-embedded/nb"
+documentation = "https://docs.rs/nb"
+readme = "README.md"
 version = "0.1.1"
+include = [
+    "/**/*.rs",
+    "/Cargo.toml",
+    "/README.md",
+    "/CHANGELOG.md",
+    "/LICENSE-MIT",
+    "/LICENSE-APACHE",
+]
```

Comment:
This is probably not a good idea since it is easy to miss stuff when publishing. I think I'd rather go with the "publish all" default.

Reply:
ok done.